### DISCUSSION TITLE
Can Map control have Parent other than Page ?

### DISCUSSION THREAD

**Discussion by UriSitFod** [flet-dev/flet#3961] [Q&A] [answered]
### Question

Hi !

When I make

    page.add(
        map.Map(
            expand=True,
            configuration=map.MapConfiguration(
                initial_center=map.MapLatitudeLongitude(15, 10),

- everything works magnificently

but when I apply

    page.add(
        ft.ResponsiveRow(
            [
                map.Map(

or

    page.add(
        ft.Container(
                map.Map(

then the map is not displayed(

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Two possibilities:
- provide constraints (`width` and `height`) to the `Map` itself or the `Container`
- `expand` the `Container`

**Reply by UriSitFod** [reply]
Thank you very much - your advice helped
